src/server/server.ts

```diff
@@ -219,7 +220,7 @@ namespace ts.server {
         }
     }
 
-    class NodeTypingsInstaller implements ITypingsInstaller {
+    export class NodeTypingsInstaller implements ITypingsInstaller {

```

Comment:
Whoops, was trying to undo the change I made to the other `NodeTypingsInstaller`.